sdk/docs/sharable/sdk/reference/daml/stdlib/Prelude.rst

```diff
@@ -1523,6 +1523,10 @@ Data Types
 
   The ``Party`` type represents a party to a contract\.
 
+  **instance** :ref:`HasFromHex <class-da-crypto-text-hasfromhex-84972>` (`Optional <type-da-internal-prelude-optional-37153_>`_ `Party <type-da-internal-lf-party-57932_>`_)
+
+  **instance** :ref:`HasToHex <class-da-crypto-text-hastohex-92431>` `Party <type-da-internal-lf-party-57932_>`_
+
```

Comment:
please mode the classes `HasFromHex` and `HasToHex` from the `DA.Text` to `DA.Crypto.Text`

Reply:
in fact move also `DA.isHex`. 
Since the feature is alpha, we may want to delete/change anything we defined for cctp. it would be easier if those function are not define in stable package.